config/locales/en.yml

```diff
@@ -974,7 +974,7 @@ en:
     orders:
       edit:
         order_sure_want_to: Are you sure you want to %{event} this order?
-        voucher_tax_included_in_price: "%{label} (tax included in price)"
+        voucher_tax_included_in_price: "%{label} (tax included in voucher)"
```

Comment:
Oh and we need to update `spec/helpers/admin/orders_helper_spec.rb:49`

Reply:
Done.